notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1916 @@
+{
```

Comment:
<p>Better link for FoldedDistribution: https://num.pyro.ai/en/stable/distributions.html#foldeddistribution</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>